src/Stratis.SmartContracts.CLR/Loader/ContractAssemblyLoader.cs

```diff
@@ -1,27 +1,47 @@
 ﻿using System;
+using System.IO;
 using System.Reflection;
+using System.Runtime.Loader;
 using CSharpFunctionalExtensions;
 
 namespace Stratis.SmartContracts.CLR.Loader
 {
+    public class SmartContractLoadContext : AssemblyLoadContext
+    {
+        private AssemblyLoadContext defaultContext;
+
+        public SmartContractLoadContext(AssemblyLoadContext defaultContext)
+        {
+            this.defaultContext = defaultContext;
+        }
+
+        protected override Assembly Load(AssemblyName assemblyName)
+        {
+            // Ensure that an exact compatible version is used.
+            if (assemblyName.Name == "Stratis.SmartContracts.Standards" && assemblyName.Version.Major == 1 && assemblyName.Version.Minor < 4)
+                return this.LoadFromAssemblyPath(Path.GetFullPath("Stratis.SmartContracts.Standards (64 bit).dll"));
+
```

Comment:
Possible to append version suffix to name, 64 bit seems sounds like compilation of dll for 32 bit vs 64 bit systems, WDYT?